packages/cluster/src/commands/up.js

```diff
@@ -48,7 +49,7 @@ const run = async function(argv) {
             cmd: 'docker-compose',
             args: [
                 '-p',
-                makeComposeProject(name),
+                name,
```

Comment:
I think it's important to keep the docker-compose project namespaced (i.e. prefixed with `d2-cluster` as it was previously), otherwise `dev` is very generic and could well mess up the `docker ps` call below (if there's any other docker container on the maching with `dev` in the name.

Reply:
Ok, will change so the docker-compose parts are prefixed.

Reply:
88e7702